### PR TITLE
Responsiveness

### PR DESCRIPTION
**Pull request by MdMajor**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,14 +1,34 @@
...
+let dob = document.getElementById("dob");
+let email = document.getElementById("email");
+let phone = document.getElementById("phone");
+let city = document.getElementById("city");
+let institute = document.getElementById("institute");
 var gen = document.getElementsByName("gender");
 var qual = document.getElementsByName("c");
 
+
+    if (name.value == 0) {
```
Comment (heyAfsar) [inline app.js:13]: what if the user enters special characters and numbers in Name?.

This needs to be validated.

File: app.js [hunk 2]
```diff
@@ -1,14 +1,34 @@
...
+        return false;
+      }
+    else if(email.value == 0) {
+        window.alert("please enter your emaili");
+        return false;
+      }
+    else{
+    
+ 
     for(let i = 0; i<gen.length; i++){
```
Comment (heyAfsar) [inline app.js:32]: by the time user reaches here, if he didn't enter valid values above, he might have seen 5-6 popups, this is a bad user experiance. 

update this as soon as you can

File: app.js [hunk 3]
```diff
@@ -1,14 +1,34 @@
...
+
+    if (name.value == 0) {
+      window.alert("please enter your name");
+      return false;
+    }
+    else if(dob.value == 0) {
+        window.alert("please enter your date of birth");
+        return false;
+      }
+    else if(phone.value == 0) {
```
Comment (heyAfsar) [inline app.js:21]: only numbers should be allowed and maximum length of 10 digits

File: app.js [hunk 4]
```diff
@@ -1,14 +1,34 @@
...
+        window.alert("please enter your contact number");
+        return false;
+      }
+    else if(email.value == 0) {
```
Comment (heyAfsar) [inline app.js:25]: email validation has to be done.

Form email input default provides these validations. use similar controls to handle validations out of the box

File: app.js [hunk 5]
```diff
@@ -45,5 +66,16 @@ var qual = document.getElementsByName("c");
       }).catch(function(error) {
         console.log(error);
       });
+    }
+
+      window.location.href="success.html";
```
Comment (heyAfsar) [inline app.js:71]: proper routing has to be used for the page navigation

File: index.html [hunk 6]
```diff
@@ -11,19 +11,19 @@
...
+            <input type="text" id="name" placeholder="Enter your name" required />
           </div>
           <div class="input-box">
             <span class="details">Date of Birth</span>
             <input type="date" id="dob" required>
           </div>
           <div class="input-box">
             <span class="details">Email</span>
-            <input type="text" id="email" placeholder="Enter your email" required>
+            <input type="text" id="email" placeholder="Enter your email" required />
```
Comment (heyAfsar) [inline index.html:26]: EMail input type should be used for default email vaildations

**Review by heyAfsar — CHANGES_REQUESTED** [review]

File: app.js (hunk 2 above)
Comment (MdMajor) [reply app.js:34]: popup comes one by one

File: app.js (hunk 3 above)
Comment (MdMajor) [reply app.js:23]: issue created: #6

File: app.js (hunk 5 above)
Comment (MdMajor) [reply app.js]: issue created: #8

File: app.js (hunk 1 above)
Comment (MdMajor) [reply app.js:15]: issue created: #10

**Review by heyAfsar — APPROVED** [review]
LGTM

**Pull request merged**